src/actions-util.ts

```diff
@@ -643,6 +647,9 @@ export async function createStatusReportBase(
       workflowStartedAt
     );
   }
+  const runnerOs = process.env["RUNNER_OS"] || "";
+  const runnerArch = process.env["RUNNER_ARCH"] || "";
```

Comment:
We can expect that these env vars are available always. This is the pattern we are using elsewhere to get env vars.

```suggestion
  const runnerOs = getRequiredEnvParam("RUNNER_OS");
  const runnerArch = getRequiredEnvParam("RUNNER_ARCH");
```